react/features/base/react/components/web/SectionList.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Container } from './index';
+
+type Props = {
+
+    /**
+     * Used to extract a unique key for a given item at the specified index.
+     * Key is used for caching and as the react key to track item re-ordering.
+     */
+    keyExtractor: Function,
+
+    /**
+     * Returns a React componet that renders each Item in the list
+     */
+    renderItem: Function,
+
+    /**
+     * Returns a React component that is the header for every section of the
+     * list
+     */
+    renderSectionHeader: Function,
+
+    /**
+     * Sections to be rendered in the following format:
+     *
+     * [
+     *   {
+     *     title: string,               <- section title
+     *     key: string,                 <- unique key for the section
+     *     data: [                      <- Array of items in the section
+     *       {
+     *         colorBase: string,       <- the color base of the avatar
+     *         title: string,           <- item title
+     *         url: string,             <- item url
+     *         lines: Array<string>     <- additional lines to be rendered
+     *       }
+     *     ]
+     *   }
+     * ]
+     */
+    sections: Array<Object>,
+
+    /**
+     * onItemClick is a function that defines what happens when
+     * an item in the section list is clicked
+     */
+    onItemClick: Function
+};
+
+/**
+ * Implements a React/Web {@link Component} for displaying a list with
+ * sections similar to React Native's {@code SectionList} in order to
+ * faciliate cross-platform source code.
+ *
+ * @extends Component
+ */
+export default class SectionList extends Component<Props> {
+    /**
+     * Renders the content of this component.
+     *
+     * @returns {React.ReactNode}
+     */
+    render() {
+
+
+        const { renderSectionHeader, renderItem, sections } = this.props;
+
+        if (sections) {
+            return (
+            /* eslint-disable no-extra-parens */
+                <Container>
+                    {
+                        this.props.sections.map((section, sectionIndex) => (
```

Comment:
Use `sections`.

Reply:
- change this.props.sections to sections